docs/src/styles/docs.css

```
@@ -0,0 +1,364 @@
+
+.sidebar-header {
+  padding: 24px 24px 16px;
+}
+
+.repo-title {
+  font-size: 18px;
+  font-weight: 700;
+  margin: 0 0 4px;
+  color: var(--text-primary);
+  letter-spacing: 3px;
+  font-family: 'Rajdhani', sans-serif;
```

Stylelint (17.9.0)

[error] Expected no quotes around "Rajdhani" (font-family-name-quotes)

(font-family-name-quotes)